docs/kpod-cp.1.md

```diff
@@ -0,0 +1,46 @@
+% kpod(1) kpod-mount - Mount a working container's root filesystem.
+% Dan Walsh
+# kpod-cp "1" "August 2017" "kpod"
+
+## NAME
+kpod-cp - Copy files/folders between a container and the local filesystem.
+
+## Description
+We chose not to implement this feature even though the upstream docker project
+has it. We have a much stronger capability.  Using standard kpod-mount and
+kpod-umount, we can take advantage of the entire linux command line, rather then
+just cp.
+
+If a user wants to copy contents out of a container or into a container, they
+can execute a few simple commands.
+
+You can copy from the container's file system to the local machine or the
+reverse, from the local filesystem to the container. 
+
+If you want to copy the /etc/foobar directory out of a container execute the
```

Comment:
I'm hemming/hawing on this.  Maybe:  "container execute" to "container and onto the /tmp directory of the local host execute".